src/main/java/org/assertj/core/internal/Paths.java

```diff
@@ -496,4 +499,22 @@ private static void assertExpectedEndPathIsNotNull(final Path end) {
     requireNonNull(end, "the expected end path should not be null");
   }
 
+  public void assertIsEmptyPath(AssertionInfo info, Path actual) {
+    assertIsRegularFile(info, actual);
+    try {
+      if (size(actual) == 0) return;
+    } catch (IOException e) {
+
```

Comment:
Also here, instead of leaving the `catch` empty a `PathsException` can be thrown:
```suggestion
      throw new PathsException(FAILED_TO_RESOLVE_ACTUAL_REAL_PATH, e);
```